common/tier4_autoware_utils/include/tier4_autoware_utils/system/backtrace.hpp

```diff
@@ -43,7 +45,7 @@ inline void print_backtrace()
   for (int i = 1; i < addrlen; i++) {
     ss << "   @   " << symbol_list[i] << std::endl;
   }
-  std::cerr << ss.str() << std::endl;
+  RCLCPP_DEBUG_STREAM(rclcpp::get_logger("tier4_autoware_utils"), ss.str());
```

Comment:
Since this function is long with a for-loop and is very unlikely to be expanded inline, can you move this to cpp file? Inline functions are compiled by the time they are included and thus has bad effect for compile time.

Reply:
Gool information, thank you! I'll address

Reply:
@soblin addressed here
https://github.com/autowarefoundation/autoware.universe/pull/5101/commits/6321dbd47f39570c29e87f685db9ff32bf5ba3ca